code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -23,6 +23,13 @@
 	. = ..()
 	. += span_notice("You need a crowbar to pry this open!")
 
+/obj/structure/largecrate/CanAllowThrough(atom/movable/mover, turf/target)
+	. = ..()
+	if(istype(mover) && CHECK_BITFIELD(mover.flags_pass, PASSTABLE))
+		return TRUE
+	var/obj/structure/S = locate(/obj/structure) in get_turf(mover)
+	if(S?.climbable && !(S.flags_atom & ON_BORDER) && climbable && isliving(mover)) //Climbable non-border objects allow you to universally climb over others
+		return TRUE
```

Comment:
???

Reply:
yeah idk what this is, I'll just delete it.